packages/generated-files-bot/test/fixtures/config/valid-config.yml

```diff
@@ -1,5 +1,8 @@
 generatedFiles:
 - 'file1.txt'
+- path: '**.html'
+- path: 'docs/**.md'
+- path: 'src/.data/**.json'
```

Comment:
nit: I might separate this config/test scenario into an explicit test case for globbing rather than overloading the basic commenting test.